docs-web/configuration/virtualserver-and-virtualserverroute-resources.md

```diff
@@ -1026,8 +1026,8 @@ return:
   type: application/json
   body: "{\"msg\": \"You don't have permission to do this\"}"
   headers:
-  - name: x-debug-original-status
-    value: ${status}
+  - name: x-debug-original-statuses
+    value: ${upstream_status}
```

Comment:
Please update the example in `ErrorPage.Return.Header` and the description in the corresponding table. (upstream status instead of status)